src/Components/Tooltip.rei

```diff
@@ -0,0 +1,16 @@
+open Oni_Core;
+open Revery.UI;
+
+module Overlay: {
```

Comment:
I like the API you designed for this (and using the `Trigger` module, but `included`, so the interface is streamlined) - seems like it'll be very easy to add in other spots, too.